### PR TITLE
Added values for distance calculation

### PR DESCRIPTION
**Pull request by thrvg**
Added some values required for distance calculation. Previously, I had put 1 as a placeholder there.

### CODE REVIEW

File: modules/targetAcquisition/taxi/Taxi.py [hunk 1]
```diff
@@ -95,16 +95,16 @@ def calculate_distance(pts):
...
         # (Known) real Height of the box in mm
         realHeight = 101.6
 
         # (Known) height of the image in pixels
-        imageHeight = 1
+        imageHeight = 1080
 
         # (Known) height of the sensor in mm
-        sensorHeight = 1
+        sensorHeight = 4.63
```
Comment (Shrinjay) [inline modules/targetAcquisition/taxi/Taxi.py:107]: Nitpick here, but would be nice if you could move these values into constant variables declared in CONSTANT_CASE (example of constant case there) at the top of the file.

**Review by Shrinjay — CHANGES_REQUESTED** [review]

File: modules/targetAcquisition/taxi/Taxi.py (hunk 1 above)
Comment (thrvg) [reply modules/targetAcquisition/taxi/Taxi.py]: Changed to CONSTANT_CASE!

**Comment by thrvg** [comment]
Hey I put the CONSTANTS at the top

**Review by Shrinjay — APPROVED** [review]
Sorry for the late review, LGTM now

**Pull request merged**